### PR TITLE
freebsd adding ucontext api

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

File: src/unix/bsd/freebsdlike/freebsd/x86_64/mod.rs [hunk 1]
```diff
@@ -301,7 +301,6 @@ cfg_if! {
         }
     }
 }
-
```
Comment (Amanieu) [inline src/unix/bsd/freebsdlike/freebsd/x86_64/mod.rs:304]: Accidentally removed this line?

**Comment by JohnTitor** [comment]
@devnexen Could you address the [CI failure](https://github.com/rust-lang/libc/runs/6067834343?check_suite_focus=true)?

**Comment by JohnTitor** [comment]
@bors try
To revive the logs...

**Comment by JohnTitor** [comment]
```
 error: the struct `#[repr(align(u16))]` attribute is experimental (see issue #33626)
   --> src/macros.rs:112:13
    |
112 |               pub struct $i { $($field)* }
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    | 
   ::: src/lib.rs
    |
93  | / cfg_if! {
94  | |     if #[cfg(windows)] {
95  | |         mod fixed_width_ints;
96  | |         pub use fixed_width_ints::*;
...   |
162 | |     }
163 | | }
    | |_- in this macro invocation

error: non-string literals in attributes, or string literals in top-level positions, are experimental (see issue #34981)
   --> src/macros.rs:111:15
    |
111 |               $(#[$attr])*
    |                 ^^^^^^^^
    | 
   ::: src/lib.rs
    |
93  | / cfg_if! {
94  | |     if #[cfg(windows)] {
95  | |         mod fixed_width_ints;
96  | |         pub use fixed_width_ints::*;
...   |
162 | |     }
163 | | }
    | |_- in this macro invocation

error: aborting due to previous error(s)
```

@devnexen Could you check?
@rustbot author

**Comment by JohnTitor** [comment]
Thank you! @bors r+

**Comment by JohnTitor** [comment]
Fails on 1.19: https://github.com/rust-lang/libc/actions/runs/4658417107/jobs/8244710144#step:5:1376

**Comment by JohnTitor** [comment]
Closing as inactive, feel free to resolve the above comment and re-submit a PR if you're still interested in this change. Thanks for your contribution anyway!

**Pull request closed without merging**